smoke-test/src/main/java/org/opennms/smoketest/containers/MinionContainer.java

```diff
@@ -241,8 +247,17 @@ public WaitForMinion(MinionContainer container) {
         @Override
         protected void waitUntilReady() {
             LOG.info("Waiting for Minion health check...");
-            final InetSocketAddress sshAddr = container.getSshAddress();
-            awaitHealthCheckSucceeded(sshAddr, 5, "Minion");
```

Comment:
Do we still need the code for `awaitHealthCheckSucceeded`?

Reply:
That's a good question.  @cgorantla what do you think?.

Reply:
May be it  will be used  in another smoke test which should still check for SSH connection on Minion/Sentinel.

Reply:
* I propose that all integration test now rely on the rest api when waiting for container startup.
* There should be an additional integration test though, that checks if the `health-check` command is available on the command line. That test could be based on the code of the current ssh-based startup check.

Reply:
Yes I agree. I'll include that in this PR